So can it be ignored from the package?

Also, the failed tests are located in:

```
@pnpm/auth.commands@1100.0.4 : .test auth/commands
  $ cross-env NODE_OPTIONS="$NODE_OPTIONS --experimental-vm-modules --disable-warning=ExperimentalWarning --disable-warning=DEP0169" jest
   ELIFECYCLE  Command failed with exit code 1.
   ELIFECYCLE  Command failed with exit code 1.
  Error: Process completed with exit code 1.
```

It doesn't seem to be due to this PR.

--- a/pnpm-workspace.yaml
+++ b/pnpm-workspace.yaml
@@ -67,6 +67,7 @@ auditConfig:
     - GHSA-76c9-3jph-rj3q
     - GHSA-ffrw-9mx8-89p8
     - GHSA-mh29-5h37-fv8m
+    - GHSA-w5hq-g745-h8pq
 
 catalog:
   '@babel/core': ^7.28.3

